lib/kernel.coffee

```diff
@@ -15,7 +15,7 @@ class Kernel
         console.log 'Kernel spec:', @kernelSpec
         console.log 'Kernel configuration:', @config
         console.log 'Kernel configuration file path:', @configPath
-        @language = @kernelSpec.grammarLanguage
+        @language = @kernelSpec.language
```

Comment:
> We need to remove this line as well.
Done in #339 

> After some more thought and since Kernel gets a grammar, I think this line should be @language = KernelManager::getLanguageFor grammar.
Fixed!